db/schema.rb

```diff
@@ -47,6 +46,7 @@
     t.date "average_views_updated_at"
     t.integer "wiki_id"
     t.integer "mw_page_id"
+    t.boolean "tracked", default: true
```

Comment:
Is this supposed to be on the "articles" table as well?

Reply:
No. Making an article tracked will make it so for every otther course irrespective of whether the instructor wants this article to be tracked or not. So it is only available on ArticlesCourses where it can be namespaced along with a course id so other courses can still make the decision of whether they want the article to be tracked or not.